news/2017-06-27-beatmap-spotlight-may-2017.md

```diff
@@ -26,96 +27,98 @@ First of all, congratulations to the winners of the April 2017 Spotlights. They
 
 [![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/black-or-white.jpg)](https://osu.ppy.sh/s/565779)
 
-Let me introduce you another extreme 7K marathon map by Kawawa. I really like how the map structure flows nicely with the song intensity. From the map itself, i can see the mapper have a really good experience in term of emphasizing the melody itself. What caught my attention is the jacking style pattern in the map which it is pretty new and interesting to see. Besides giving a great pattern usage, the speed change through the map are astonishing extreme and fun. The mapper really did a good job on hightlighting certain beats magnitude with a suitable slider velocity. By advancing through the map, player could easily experience the consciousness of the music along with the note arrangement emphasized by the mapper intention. Have a try while you are on it.
+Let me introduce you another extreme 7K marathon map by **[Kawawa](https://osu.ppy.sh/users/4647754)**. Pleasure to the eye how the map structure flows nicely with the song intensity. He has a really good experience in term of emphasizing the melody itself. What caught my attention is the jacking style pattern in the map which it is pretty new and interesting to see. Besides giving a great pattern usage, the speed change through the map are astonishing extreme and fun.
+
+He did a good job on highlighting certain beats magnitude with a suitable slider velocity. By advancing through the map, player could easily experience the consciousness of the music along with the note arrangement emphasized by the mapper intention. Have a try while you are on it.
 
 [![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/shine.jpg)](https://osu.ppy.sh/s/456289)
 
-For his 5th ranked map, Frim4503 offers us another 5K marathon, this time for a drum & bass love live opening remix !
-As usual we have a mix of LN and chords playing really good, making it a really fun map to try big accuracy scores. SV are really lightly used aswell with a quite unique style, focussing on the voice before each kiai. Without changing much of music/mapping style, Frim4503 show us one more time that anti-meta is not dead with a really accessible and catchy map.
+For his 5th ranked map, **[Frim4503](https://osu.ppy.sh/users/2945775)** offers us another 5K marathon, this time for a drum & bass love live opening remix!
```

Comment:
``this time for`` -> this time with
``opening`` -> OP
``love live`` -> Love Live